openssl/src/rsa.rs

```diff
@@ -597,6 +597,25 @@ impl Rsa<Private> {
         }
     }
 
+    /// Generates a public/private key pair with the specified size.
+    ///
+    /// This corresponds to [`RSA_generate_key_ex`].
+    ///
+    /// [`RSA_generate_key_ex`]: https://www.openssl.org/docs/man1.1.0/crypto/RSA_generate_key_ex.html
+    pub fn generate_with_e(bits: u32, e: BigNum) -> Result<Rsa<Private>, ErrorStack> {
```

Comment:
This can just take a `&BigNumRef`.

Reply:
I don't think so because the `ffi` takes `e: *mut BIGNUM`

```rust
pub fn RSA_generate_key_ex(
        rsa: *mut RSA,
        bits: c_int,
        e: *mut BIGNUM,
        cb: *mut BN_GENCB,
    ) -> c_int;
```

Reply:
You can get that from a `&BigNumRef` with `.as_ptr()`.

Reply:
`BigNum` and `BigNumRef` are a bit like `String` and `str` or `PathBuf` and `Path`.

https://github.com/sfackler/rust-openssl/blob/master/openssl/src/rsa.rs#L605-L607